recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc

```diff
@@ -304,6 +304,7 @@ FILES_libegl-imx-dev = "${libdir}/libEGL${SOLIBSDEV} ${includedir}/EGL ${include
 FILES_libgal-imx = "${libdir}/libGAL${SOLIBS} ${libdir}/libGAL_egl${SOLIBS}"
 FILES_libgal-imx-dev = "${libdir}/libGAL${SOLIBSDEV} ${includedir}/HAL"
 RDEPENDS_libgal-imx += "kernel-module-imx-gpu-viv"
+RDEPENDS_libgal-imx += "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'valgrind', '', d)}"
```

Comment:
Why is this dependant on `wayland` support?

Reply:
Only because we're focusing on Wayland, I can remove the dependency.

Reply:
Oh, actually I didn't notice the next line is an `RSUGGESTS` for `valgrind`. Seems that doesn't pull in `valgrind`.  Not supported by `rpm`?

Reply:
Sorry @thochstein but we cannot accept that; we must be careful and support any use case customer might have (except if there is a real reason to not do) and I think there is no reason to not be generic here.

Reply:
One question which comes to my mind is what `libgal-imx` stands for? the `valgrind` support seems to qualify as an optional support and thus as a `PACKAGECONFIG[valgrind] = ...` thing. Am I wrong?

Reply:
I'll talk to Prabhu about it tomorrow. I guess you ask about `libgal-imx` because we cannot use `PACKAGECONFIG` for a specific non-main package?

Reply:
Your `RDEPENDS` were in `libgal-imx` hence the question. We can add a `PACKAGECONFIG` and check if it is enabled byhand and use it to add it to the sub-packages.

Reply:
Oh, right, got it now. I think it's a good idea.